uk.co.compucorp.civicrm.hrleaveandabsences/api/v3/LeaveRequest.php

```diff
@@ -150,6 +158,15 @@ function _civicrm_api3_leave_request_getfull_spec(&$spec) {
     'type'         => CRM_Utils_Type::T_BOOLEAN,
     'api.required' => 0
   ];
+
+  $spec['unassigned'] = [
+    'name' => 'unassigned',
+    'title' => 'Unassigned only?',
+    'description' => 'Include only Leave Requests of contacts without active leave managers?',
+    'type' => CRM_Utils_Type::T_BOOLEAN,
+    'api.default' => FALSE,
```

Comment:
Please remove this. It will set unassigned to false when unassigned is not present